jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2025,7 +2163,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "[{\"webserverId\":{{WebserverId}},\"webserverName\":{{HealthCheckWebServer}},\"hostName\":\"{{host}}\",\"portNumber\":\"80\",\"httpsPort\":\"443\",\"statusPath\":\"/apache_pb.png\",\"groupIds\":[{\"groupId\":\"1\"},{\"groupId\":\"2\"},{\"groupId\":{{GroupId}}}]}]"
+							"raw": "[{\"webserverId\":{{WebserverId}},\"webserverName\":\"Postman-test-webserver2\",\"hostName\":\"{{host}}\",\"portNumber\":\"80\",\"httpsPort\":\"443\",\"statusPath\":\"/apache_pb.png\",\"groupIds\":[{\"groupId\":\"1\"},{\"groupId\":\"2\"},{\"groupId\":{{GroupId}}}]}]"
```

Comment:
The web server name needs to be a an environment variable.

Also, should the group IDs be hard coded?